src/applications/personalization/profile/components/connected-apps/AppDeletedAlert.jsx

```diff
@@ -16,7 +16,7 @@ export const AppDeletedAlert = props => {
       href={privacyUrl}
       target="_blank"
       rel="noopener noreferrer"
-      aria-label={`${title} privacy policy`}
+      aria-label={`Review the ${title} privacy policy`}
```

Comment:
I think a better solution for this is to make `review the ${title} privacy policy` the text of the anchor link itself. There doesn't seem to be a reason for the clickable text to only be "privacy policy."